geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/ClientUserAuths.java

```diff
@@ -78,6 +80,10 @@ public UserAuthAttributes getUserAuthAttributes(long userId) {
     return uniqueIdVsUserAuth.get(userId);
   }
 
+  protected Collection<Subject> getSubjects() {
```

Comment:
probably mark this as @VisibleForTesting

Reply:
Thanks for the suggestion. Fixed.